chart/templates/rbac.yaml

```diff
@@ -3,93 +3,27 @@ kind: ClusterRole
 metadata:
   name: {{ .Release.Name }}
 rules:
-- apiGroups:
-    - management.cattle.io
-  resources:
-    - 'settings'
-  verbs:
-    - 'get'
-    - 'watch'
-    - 'list'
-- apiGroups:
-  - ""
-  resources:
-  - 'events'
-  verbs:
-  - 'patch'
-  - 'create'
-- apiGroups:
-  - ""
-  resources:
-  - 'secrets'
-  verbs:
-  - 'get'
-  - 'watch'
-  - 'list'
-  - 'create'
-  - 'update'
-  - 'patch'
-  - 'delete'
-- apiGroups:
-  - ""
-  resources:
-  - 'pods'
-  - 'serviceaccounts'
-  verbs:
-  - 'get'
-  - 'create'
-  - 'delete'
-  - 'list'
-  - 'watch'
-- apiGroups:
-    - "rbac.authorization.k8s.io"
-  resources:
-    - 'rolebindings'
-    - 'roles'
-  verbs:
-    - 'get'
-    - 'create'
-    - 'delete'
-    - 'watch'
-- apiGroups:
-  - ""
-  resources:
-  - 'pods/log'
-  verbs:
-  - 'get'
-- apiGroups:
-  - elemental.cattle.io
-  resources:
-  - '*'
-  verbs:
-  - '*'
-- apiGroups:
-  - fleet.cattle.io
-  resources:
-  - 'bundles'
-  verbs:
-  - '*'
-- apiGroups:
-  - apiextensions.k8s.io
-  resources:
-  - customresourcedefinitions
```

Comment:
I wonder what was this for...

Reply:
https://github.com/rancher/elemental-operator/blob/main/pkg/operator/operator.go#L58
we create CRDs in the code...

Reply:
It seems that it is some kind of leftover, because elemental-operator doesn't define any CRD

Reply:
good point, thank you, I have added it again